pages/moods-map.js

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState, useRef } from "react";
+import useSWR from "swr";
+import experienceAnalyser from "@/utils/ExperienceAnalyser";
+import NavButton from "@/components/NavButton/NavButton";
+import { Navigation, Container } from "@/components/Layout/Layout.styled";
+import styled from "styled-components";
+import Circle from "@/components/Circle/Circle";
+import { useRouter } from "next/router";
+import { motion } from "framer-motion";
+import Link from "next/link";
+
+export default function App() {
+  const router = useRouter();
+  const [screenSize, setScreenSize] = useState({ width: 0, height: 0 });
+  const containerRef = useRef(null);
+
+  useEffect(() => {
+    const handleResize = () => {
+      const containerWidth = containerRef.current
+        ? containerRef.current.offsetWidth
+        : window.innerWidth;
+      const containerHeight = containerRef.current
+        ? containerRef.current.offsetHeight
+        : window.innerHeight;
+      setScreenSize({ width: containerWidth, height: containerHeight });
+    };
+
+    handleResize();
+    window.addEventListener("resize", handleResize);
+    return () => window.removeEventListener("resize", handleResize);
+  }, []);
+
+  const { data, isLoading } = useSWR("/api/entries");
+  if (isLoading) {
+    return <p>loading...</p>;
+  }
+  if (!data) {
+    return <p>no connection to database</p>;
+  }
+
+  const result = experienceAnalyser(data);
+
+  return (
+    <>
+      <Container>
+        <motion.div animate={{ opacity: 0 }} transition={{ delay: 8 }}>
+          <DragInfo>
+            {result[1].totalCount} entries in our collection at the moment.
+            <br />
+            click and drag the spheres
+          </DragInfo>
+        </motion.div>
+        <Grid ref={containerRef}>
+          {result[0].map((entry, index) => (
```

Comment:
if you only need entry 0 and 1 from results you can use Array destructuring for better readability:
const [resultCount, resultList] = experienceAnalyser(data);